doc/source/conda-env.yml

```diff
@@ -1,5 +1,8 @@
 name: libtiledb-env
+channels:
+  - conda-forge
+  - defaults
 dependencies:
   - python=3.4
```

Comment:
can this be updated to 3.6 (tiledb-py won't work on python3.4 at the moment)?

Reply:
Yes I'll update it (though this is just for RTD).